portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMFileEntry.testcase

```diff
@@ -2568,12 +2568,80 @@ definition {
 	}
 
 	@description = "This is a test for LPS-145630. It checks that the PDFs linked in web articles content can be updated."
-	@ignore = "Test Stub"
 	@priority = "4"
 	test PDFLinkedInWCCanBeUpdated {
+		JSONDocument.addFileWithUploadedFile(
+			dmDocumentTitle = "Document_1.pdf",
+			groupName = "Guest",
+			mimeType = "application/pdf",
+			sourceFileName = "Document_1.pdf");
+
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "guest");
+
+		WebContentNavigator.gotoAddCP();
+
+		PortletEntry.inputTitle(title = "Web Content Title");
+
+		CKEditor.addTextWithLink(
+			displayText = "Web Content Content",
+			dmDocumentTitle = "Document_1.pdf",
+			fieldLabel = "content");
+
+		PortletEntry.publish();
+
+		JSONLayout.addPublicLayout(
+			groupName = "Guest",
+			layoutName = "Test Page");
+
+		JSONLayout.addWidgetToPublicLayout(
+			groupName = "Guest",
+			layoutName = "Test Page",
+			widgetName = "Web Content Display");
 
-		// TODO LPS-149475 PDFLinkedInWCCanBeUpdated pending implementation
+		Navigator.gotoSitePage(
+			pageName = "Test Page",
+			siteName = "Guest");
+
+		WebContentDisplayPortlet.selectWebContent(webContentTitle = "Web Content Title");
+
+		IFrame.closeFrame();
+
+		Click(
+			key_portletName = "Web Content Display",
+			locator1 = "Portlet#BODY",
+			value1 = "Web Content Content");
+
+		AssertLocation.assertPartialLocation(value1 = "Document_1.pdf");
+
+		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+
+		DMDocument.editCP(
+			dmDocumentFileEdit = "large_file.pdf",
+			dmDocumentFileNameEdit = "large_file.pdf",
+			dmDocumentTitle = "Document_1.pdf",
+			dmDocumentTitleEdit = "large_file.pdf");
+
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "guest");
+
+		WebContentNavigator.gotoEditCP(webContentTitle = "Web Content Title");
+
+		CKEditor.addTextWithLink(
+			displayText = "Web Content Content",
+			dmDocumentTitle = "large_file.pdf",
+			fieldLabel = "content");
+
+		PortletEntry.publish();
```

Comment:
Hey @jaydawu As we just talked about, we don't need to update the web content link after updating the document. We just need to make sure that the linked content in the article is changed after updating the uploaded file. Please help to update it and resend it, thanks.